src/main/java/uk/gov/hmcts/cmc/claimstore/services/pilotcourt/PilotCourtService.java

```diff
@@ -47,6 +47,8 @@
     private final String dataSource;
     private final AppInsights appInsights;
 
+    public static final String OTHER_COURT_ID = "OTHER";
```

Comment:
Is this relevant to string here? If yes please use same const.

https://github.com/hmcts/cmc-claim-store/blob/master/src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/OrderCreator.java#L220